src/common/appdb/Connection.ts

```diff
@@ -34,11 +33,11 @@ export default class Connection {
       subscriptions: [
```

Comment:
I checked and passwords has been saved without encryption. I will change to subscribers and check if it resolves the issue, but maybe we will need some migration to encrypt already saved connections

Reply:
I had to do the migration to fix this, now passwords are being saved encrypted at the database. I will open another PR because its a "critical" bug and can be merged already without typescript.

Reply:
https://github.com/beekeeper-studio/beekeeper-studio/pull/322

Reply:
you rock!